doc/manual/manual.tex

```diff
@@ -8485,23 +8488,249 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 
 We can observe that when the two-layer viscosity model is added, there is only one apparent peak for each case. The first peaks of the 0.01 viscosity contrast tests appear earlier and are larger in magnitude than those of 0.1 viscosity contrast tests.  There are no secondary plumes and the whole system tends to reach stability after around 500 time steps.
 
+
+\paragraph{Computation of the van Keken Problem with the Volume-of-Fluid Interface Tracking Method}
+\label{paragraph:van-keken VOF interface tracking}
+
+One can also model the van Keken problem with the volume-of-fluid (VOF) interface tracking 
+algorithm in ASPECT.
+In fact, this problem is particularly well-suited to being computed with the VOF method, since 
+it consists of two distinct, immiscible fluids and interface tracking algorithms are 
+specifically designed not to allow the two fluids to mix.
+In particular, assuming the computation is sufficiently well-resolved, the fluids will not mix 
+at sub-grid scales over the entire duration of the computation.
+However note that this implies that all computations of the van Keken problem made with the VOF 
+method must necessarily be with discontinuous initial conditions.
+Finally, since one is often interested in a high resolution image of the shape of the interface 
+between the two fluids, one can use the output of the VOF method to examine the interface 
+within individual cells or within regions consisting of groups of cells.
+
+Another advantage the VOF method has over modeling moving interface problems with compositional 
+fields is that for problems in which the interface occupies a relatively small part of the 
+computational domain, all of the computational work in the VOF method is done in cells that lie 
+in a neighborhood of the interface, rather than in the entire computational domain. 
+
+\begin{figure}[b!]
+   \centering
+\includegraphics[width=0.5\textwidth]{cookbooks/benchmarks/vof-van-keken/rms_vel_comparison.png}
+   \caption{Evolution of the root mean square velocity as a function of time for computations 
+    of the Van Keken problem made with the VOF interface tracking algorithm with five 
+    different global mesh refinements.
+    Since the VOF initial conditions are discontinuous, the above results should really be 
```

Comment:
Perhaps no need to start this sentence on a new line?

Reply:
I had been adding linebreaks at sentence ends in my new additions relatively frequently due to the fact that latex treats that type of linebreak as standard whitespace with the intent of making it easier on me to look at the diffs.

Reply:
Gotcha. Before merging, please reorganize the text into standard paragraphs and line paragraphs.

Reply:
My understanding and testing indicates that LaTeX does not insert a paragraph break unless there is a blank line, os the current approach does not actually produce malformed output (as I indicated it combines the sentences into a paragraph as usual), which was part of the reason I used the approach in question. I can do the reformatting as a large block commit as the final change before merging though.